mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -200,13 +190,12 @@ def connect(self, port, addr='localhost', uid=UNKNOWN_UID,
         """
         self._counter = self._id_counter()
         try:
-            self._conn = socket.create_connection((addr, port),
+            self._conn = socket.create_connection(('127.0.0.1', self.host_port),
```

Comment:
Just trying to skip an extra dns lookup... curious what the difference is between mac and linux?